cmd/dcrinstall/util.go

```diff
@@ -495,10 +495,10 @@ func seenBefore(bundle string) bool {
 	switch {
 	case strings.HasSuffix(bundle, ".zip"):
 		return exists((filepath.Join(destination,
-			strings.TrimRight(b, ".zip"))))
+			strings.TrimSuffix(b, ".zip"))))
```

Comment:
TrimRight's second arg is a cut set of characters, so it could trim more than desired.